crates/codegen/parser/runtime/build.rs

```diff
@@ -1,4 +0,0 @@
-fn main() {
```

Comment:
> for example, we never build codegen_parser_runtime into cdylib but only use the source inside

I suggest keeping this. While we don't build the NAPI code here for shipping, we do compile the NAPI types when we compile this crate, so I want to make sure `napi_build` runs correctly, as this affects `cargo check`, and we want to catch any issues early on before codegen runs.